go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/ahmetb/gen-crd-api-reference-docs v0.1.5
 	github.com/coreos/etcd v3.3.17+incompatible
 	github.com/coreos/go-systemd v0.0.0-20190719114852-fd7a80b32e1f
-	github.com/gardener/etcd-druid v0.1.12
+	github.com/gardener/etcd-druid v0.1.15
```

Comment:
Can we have these dependency updates in another PR?

Reply:
I don't see anything wrong but..